### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by juanpasolano** [vercel/vercel#6814] [Help] [answered]
Hi there,
We are having issues with the Vercel SSL generation.
Its been stuck "Generating SSL Certificate" for more than 10 hours now, and some users are getting the `ERR_SSL_PROTOCOL_ERROR` and cant access the site.
CNAME is pointed correctly and [CAA name is correctly returning](https://vercel.com/support/articles/domain-not-generating-ssl-certificate) a `0 issue "letsencrypt.org"`.
- CAA https://www.whatsmydns.net/#CAA/gettheplan.adp.com
- CNAME: https://www.whatsmydns.net/#CNAME/gettheplan.adp.com

What could be happening? What should I do?

**Comment by fu-sen** [comment, marked as answer]
The `adp.com` subdomain is heavily used.:
https://crt.sh/?q=adp.com

This domain is not on the [Public Suffix List](https://publicsuffix.org/).

Therefore, it is possible that the issue was rejected due to the rate limit of Let's Encrypt:
https://letsencrypt.org/docs/rate-limits/

To resolve this, `adp.com` provider action is required. This is not a Vercel level issue.

**Reply by leerob** [reply]
We've been in contact with the team, this is being resolved 😄
